x-pack/plugins/observability_solution/slo/server/services/find_slo.ts

```diff
@@ -27,7 +27,8 @@ export class FindSLO {
       params.kqlQuery ?? '',
       params.filters ?? '',
       toSort(params),
-      toPagination(params)
+      toPagination(params),
+      params.hideStale
```

Comment:
This hideStale param and the kqlFilter changes the summary document query, can we add a few tests that asserts the summary query is built proplery:
- when hideStale is true, stale filter is applied"
- when hideStale is true but kqlQuery contains summaryUpdatedAt, no stale filter is applied

Reply:
will do